app/gui2/src/stores/visualization.ts

```diff
@@ -0,0 +1,279 @@
+import { fileName } from '@/util/file'
+import Compiler from '@/workers/visualizationCompiler?worker'
+import * as vue from 'vue'
+import * as vueUseCore from '@vueuse/core'
+
+import { defineStore } from 'pinia'
+
+const moduleCache: Record<string, any> = { vue, '@vueuse/core': vueUseCore }
+// @ts-expect-error
```

Comment:
it *could*, but i decided against it because it should not be used anywhere else (other than here and the viz compiler). i should probably add that to the `@ts-expect-error` comment - although i don't have anything against adding it to `env.d.ts`